backend/app/Http/Controllers/Api/CoachingApiController.php

```
@@ -0,0 +1,20 @@
+use App\Http\Controllers\Controller;
+use App\Models\Coach;
+
+class CoachingApiController extends Controller
+{
+
+    public function getCoaches()
+    {
+        return Coach::paginate();
+    }
+
+    public function getCoach($id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (Naming Rules)

(ShortVariable)